### PR TITLE
orchestrator: Collect all node resources capacity

### PR DESCRIPTION
**Pull request by aureleoules**
### What does this PR do?

https://datadoghq.atlassian.net/browse/CAP-1582

This pull request collects all the node's resources capacities and allocatable.
We used to only collect the `supportedResources` and then set the appropriate unit in milli or the default.
Now we collect all the possible resources and set the value as milli if specified or the default unit.

### Motivation

We want to collect capacities such as GPUs (`nvidia.com/gpu`, `amd.com/gpu`, etc.). There are a lot of possible resource names and we cannot list them all so we will collect all of them.

### CODE REVIEW

**Review by fisherevans — APPROVED** [review]

File: releasenotes/notes/collect-node-capacities-a4fa1256cba0f172.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Collect all of node resource capacities and allocatable.
```
Comment (alai97) [inline releasenotes/notes/collect-node-capacities-a4fa1256cba0f172.yaml:4]:
```suggestion
    Collect all node resource capacities and allocatable metrics for cluster management.
```

**Review by alai97 — APPROVED** [review]
Small nit on the release note.

File: releasenotes/notes/collect-node-capacities-a4fa1256cba0f172.yaml (hunk 1 above)
Comment (aureleoules) [reply releasenotes/notes/collect-node-capacities-a4fa1256cba0f172.yaml]: Thanks I fixed it

**Pull request merged**